### ISSUE TITLE
C-f , C-b don't work as documented

### ISSUE THREAD

**Issue by hsanjuan** [atlas-engineer/next#635]
**Describe the bug**

Apparently the :emacs keymap is not default anymore and there is a :cua keymap taking precedence. C-f now means "search" and C-b does not work.

I think this is probably a documentation bug. I'm also a bit at lost on how to restore the original "emacs" keybindings (examples show how to make a custom keymap, or how to switch to the Vi one, but not how to restore the emacs keymap to make `C-s s` be search and `C-b` be "history backwards"). An example of what to put in my init would be great.

**Comment by jmercouris** [comment]
It is indeed a bit of a documentation bug, sorry about that! it should look like this:

```
(in-package :next)
(defclass my-buffer (gtk-buffer)
  ((keymap-scheme-name
    :accessor keymap-scheme-name
    :initarg :keymap-scheme-name
    :initform scheme:emacs
    :type keymap:scheme
    :documentation "The keymap scheme that will be used
for all modes in the current buffer.")))

(setf *buffer-class* 'my-buffer)
```

**Comment by jmercouris** [comment]
Also for reference, CUA
https://en.wikipedia.org/wiki/IBM_Common_User_Access

basically "standard" hotkeys

**Comment by Ambrevar** [comment]
I've temporarily fixed the doc issue:
e3c36c729d897af2947d43d322e802a19f871aa7.

We need a more future-proof configuration style, let's discuss it in
another thread.

**Comment by Ambrevar** [comment]
@hsanjuan Feel free to reopen if you have more questions! :)

**Comment by hsanjuan** [comment]
> It is indeed a bit of a documentation bug, sorry about that! it should look like this:
> 
> ```
> (in-package :next)
> (defclass my-buffer (gtk-buffer)
>   ((keymap-scheme-name
>     :accessor keymap-scheme-name
>     :initarg :keymap-scheme-name
>     :initform scheme:emacs
>     :type keymap:scheme
>     :documentation "The keymap scheme that will be used
> for all modes in the current buffer.")))
> 
> (setf *buffer-class* 'my-buffer)
> ```

Thank you! Works.

**Comment by hsanjuan** [comment]
> I've temporarily fixed the doc issue: [e3c36c7](https://github.com/atlas-engineer/next/commit/e3c36c729d897af2947d43d322e802a19f871aa7). We need a more future-proof configuration style, let's discuss it in another thread.

You commented on the wrong issue (fixed a different doc issue #634 ).

**Comment by Ambrevar** [comment]
Oops, you are right! :)

**Comment by Ambrevar** [comment]
Fixed in 7827dd85394c26313f7f031b3997a5cc0a847987, for real this time.
